source/mapping-rules/schema-mapping.txt

```diff
@@ -1,3 +1,137 @@
 ==============
 Schema Mapping
-==============
\ No newline at end of file
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Schema mapping is the process by which you decide how your source relational schema 
+should be represented in your target MongoDB database. You facilitate the schema 
+mapping process in Relational Migrator through customized mappings rules.
+
+Mapping Rules
+-------------
+
+During the schema mapping design process, Relational Migrator will automatically derive 
+the source data model from the schema of your relational database. The destination data 
+model can be influenced by customizing:
+
+- Mapping Rule options
+
+- Project Id field options
+
+.. image:: /img/schema-mapping/schema-mapping-relational-migrator-concept-diagram-b.png
+   :figwidth: 450 px
+   :alt: Schema Mapping Concept
+
+Example
+-------
+
+This provides a scenario and example implementation for the schema mapping process. 
+This example will be converting a relational data model to a MongoDB data model.
+
+Using the ``MongoEnterprises`` relational database, the goal is to flatten 
+all of the databases tables so that all the order and customer data will be 
+available in a single MongoDB collection.
+
+Use-Case Denormalizing Table Structure
```

Comment:
(nested sections) 
I _think_ the rest of the sections should be nested.

Reply:
Done.